convert2rhel/unit_tests/pkghandler_test.py

```diff
@@ -79,13 +80,26 @@ def __call__(self, cmd, print_cmd=True, print_output=True):
             self.called += 1
             return self.output, self.ret_code
 
+    @unit_tests.mock(utils, "run_subprocess", RunSubprocessMocked())
+    @unit_tests.mock(tool_opts, "disablerepo", ['rhel-7-extras-rpm'])
+    def test_call_yum_cmd_with_disablerepo(self):
+        pkghandler.call_yum_cmd("install")
+
+        self.assertEqual(utils.run_subprocess.cmd, "yum install -y --disablerepo=rhel-7-extras-rpm")
+
+    @unit_tests.mock(utils, "run_subprocess", RunSubprocessMocked())
+    def test_call_yum_cmd_without_disablerepo(self):
+        pkghandler.call_yum_cmd("install")
+
+        self.assertEqual(utils.run_subprocess.cmd, "yum install -y --disablerepo=*")
```

Comment:
Thinking about it, we can leave this to `test_call_yum_cmd` which covers this already. `test_call_yum_cmd_with_disablerepo` can remain
```suggestion
```